core/src/main/java/com/onewelcome/core/omisdk/handlers/TwoStepRegistrationRequestHandler.kt

```diff
@@ -0,0 +1,53 @@
+package com.onewelcome.core.omisdk.handlers
+
+import com.onegini.mobile.sdk.android.handlers.action.OneginiCustomTwoStepRegistrationAction
+import com.onegini.mobile.sdk.android.handlers.request.callback.OneginiCustomRegistrationCallback
+import com.onegini.mobile.sdk.android.model.entity.CustomInfo
+import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.flow.SharedFlow
+import kotlinx.coroutines.flow.asSharedFlow
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class TwoStepRegistrationRequestHandler @Inject constructor() : OneginiCustomTwoStepRegistrationAction {
+  private var registrationCallback: OneginiCustomRegistrationCallback? = null
+  private val _startTwoStepInputFlow = MutableSharedFlow<TwoStepInputData>(replay = 1)
+  val startTwoStepInputFlow: SharedFlow<TwoStepInputData> = _startTwoStepInputFlow.asSharedFlow()
+  var optionalData: String = ""
+
+  override fun initRegistration(
+    callback: OneginiCustomRegistrationCallback, customInfo: CustomInfo?
```

Comment:
The corresponding parameter in the supertype 'OneginiCustomTwoStepRegistrationAction' is named 'registrationCallback'. This may cause problems when calling this function with named arguments.

rename please